torch/nn/parallel/distributed.py

```diff
@@ -391,7 +391,7 @@ def _sync_params(self):
                                          self.broadcast_bucket_size)
             for tensors, module_params_data in zip(result[1:], self.modules_params_data[1:]):
                 for tensor, param_data in zip(tensors, module_params_data):
-                    param_data.set_(tensor)
```

Comment:
@kmatzen Can you elaborate on why storage could no longer be changed here?  I don't see tensor.data is called on this one